### ISSUE TITLE
How to run with camera.

### ISSUE THREAD

**Issue by CommandoHuan** [Zhongdao/Towards-Realtime-MOT#206]
Can anyone show me how to run with camera or webcam? Many thanks.

**Comment by CommandoHuan** [comment]
One more question. Can we track traffic vehicle with this?

**Comment by mheriyanto** [comment]
> Can anyone show me how to run with camera or webcam? Many thanks.

For webcam, you just run command below:

```
python demo.py --input-video /dev/video0 --weights path/to/model/weights
               --output-format video --output-root path/to/output/root
```

**Comment by mheriyanto** [comment]
> One more question. Can we track traffic vehicle with this? And if yes then how to do that?

Yes, we can traffic vehicle but we have to train [pre-trained](https://github.com/Zhongdao/Towards-Realtime-MOT#pretrained-model-and-baseline-models) model using vehicle objects.

**Comment by fridayhoho** [comment]
> > Can anyone show me how to run with camera or webcam? Many thanks.
> 
> For webcam, you just run command below:
> 
> ```
> python demo.py --input-video /dev/video0 --weights path/to/model/weights
>                --output-format video --output-root path/to/output/root
> ```

what's the path for osx mac ?    /dev/video0
thank you

**Comment by CommandoHuan** [comment]
> > Can anyone show me how to run with camera or webcam? Many thanks.
> 
> For webcam, you just run command below:
> 
> ```
> python demo.py --input-video /dev/video0 --weights path/to/model/weights
>                --output-format video --output-root path/to/output/root
> ```

Thank you very much.
